### DISCUSSION TITLE
Control the balance per audio device?!

### DISCUSSION THREAD

**Discussion by WoozyFace** [MonitorControl/MonitorControl#1230] [General]
### Before opening the issue, have you...?

- [x] Searched for existing issues

### Is your feature request related to a problem? Please describe

I have two monitors on my Mac mini now I want to hear only the left channel on the left screen and the right channel on the right screen. If apple did not gray out stuff in audio/midi devices, we could set the slider for the appropriate channel.

### Describe the solution you'd like

The ability to control the left and right volume per screen/output device.

### Describe alternatives you've considered

There is no alternative.

### Anything else?

Thanks!

**Comment by waydabber** [comment]
Unfortunately there is no DDC command to control this (there is `audioBalanceLR` 0x93 but it is not really supported by any display maker afaik).

**Reply by WoozyFace** [reply]
Okay understand that there is no DDC command, but how about the per channel volume slider per output device, like see image?
![Scherm­afbeelding 2022-11-18 om 23 51 05](https://user-images.githubusercontent.com/91032578/202816222-0479a5bc-144f-497d-8605-da99c0f48e5d.png)

**Reply by waydabber** [reply]
Yes, I understand that, but macOS does not support volume control for digital audio outputs (unlike Windows for example). You need a virtual audio device as output that has an audio volume filter and then this virtual device can output the audio to the real audio out. This is out of scope for this project, but there are solutions that do this.
